core/core.go

```diff
@@ -226,6 +228,12 @@ func (n *IpfsNode) startOnlineServicesWithHost(ctx context.Context, host p2phost
 	// setup name system
 	n.Namesys = namesys.NewNameSystem(n.Routing)
 
+	// setup ipns republishing
+	n.IpnsRepub = ipnsrp.NewRepublisher(n.Routing, n.Peerstore)
+	n.IpnsRepub.AddName(n.Identity)
+
+	n.Process().Go(n.IpnsRepub.Run)
```

Comment:
idk, maybe `NewRepublisher` should launch this, or have:
```go
n.IpnsRepub.Run(n.Process()) // calls proc.Go(func) inside.
```

seems like too low level ctrl